packages/intl/src/strings.ts

```diff
@@ -1982,6 +1982,8 @@ All attachments will be downloaded & cached again on access.
       ]()}`
     }),
   backupReadyToDownload: () => t`Your backup is ready to download`,
+  vaultLocked: () => t`Vault locked`,
```

Comment:
```suggestion
```

Reply:
This string is used here:

https://github.com/streetwriters/notesnook/pull/6830/files#diff-45c664ba81d530ce7480cf6bf8ad6d3680aa845598589257607979adc7b3c169R190

Reply:
Oops. Undoing...